### DISCUSSION TITLE
插件下载的问题

### DISCUSSION THREAD

**Discussion by rose8928** [opensumi/core#2155] [Q&A]
如何实现通过脚本下载marketplace.visualstudio.com网站下的插件，我模仿download-extensions.js文件自己实现了一下，报错lua-3.6.4-win32-x64 插件安装失败: Response code 429 (Too Many Requests)

**Comment by erha19** [comment]
看报错像是接口存在调用限制，超额了

**Comment by rose8928** [comment]
但是只有一个下载调用，就是模仿download-extensions.js文件写的，而且第一个连接不行，第二个连接就可以，但是我要下载win64版本，第二个下载是默认arm的
https://marketplace.visualstudio.com/_apis/public/gallery/publishers/sumneko/vsextensions/lua/3.6.4/vspackage?targetPlatform=win32-x64
https://marketplace.visualstudio.com/_apis/public/gallery/publishers/sumneko/vsextensions/lua/3.6.4/vspackage
后面有参数就不可以，这是为什么呢

**Reply by erha19** [reply]
这种问题你得问 `marketplace.visualstudio.com` 开发人员，找一下是否有相关文档以及是否允许这样使用。

**Comment by rose8928** [comment]
好的
